src/components/Dashboard/StaffNotation.vue

```diff
@@ -145,6 +146,18 @@
                 </v-list-item-title>
               </v-list-item>
             </template>
+
+            <template v-if="isBenBcCccUlc && !isHistorical">
+              <v-list-item
+                data-type="consent-continue-out"
+                @click="goToConsentContinuationOutFiling()"
+                :disabled="disabled || isAdminFreeze"
+              >
+                <v-list-item-title>
+                  <span class="app-blue">Consent to Continuation Out</span>
+                </v-list-item-title>
+              </v-list-item>
+            </template>
```

Comment:
Per Ethan, move this item to the bottom of the list, and yes, it's "consent to" not "consent for".